addons/beehave/nodes/beehave_tree.gd

```diff
@@ -149,7 +155,22 @@ func _exit_tree() -> void:
 		Performance.remove_custom_monitor(_process_time_metric_name)
 		BeehaveGlobalMetrics.unregister_tree(self)
 
+	BeehaveEditorDebugger.unregister_tree(get_instance_id())
+
 
 # Called by the engine to profile this tree
 func _get_process_time_metric_value() -> float:
 	return _process_time_metric_value
+
+
+func _get_debugger_data(node: Node) -> Dictionary:
+	var data := { path = node.get_path(), name = node.name, type = node.get_class_name(), id = str(node.get_instance_id()) }
```

Comment:
It seems that this method is not executed only on this node (`BeehaveTree`) but on **every** node. For example, when running it on my game it complains that some completely unrelated node does not have the method called `get_class_name()`. This might be a bug with Godot? Anyways, I solved this by adding the following infront of the method:
```suggestion
	if not node is BeehaveTree and not node is BeehaveNode:
		return {}
	var data := { path = node.get_path(), name = node.name, type = node.get_class_name(), id = str(node.get_instance_id()) }
```